tests/TestCase/Model/Table/SentencesTableTest.php

```diff
@@ -1512,4 +1513,40 @@ function testGetSeveralRandomIds() {
 
         $this->assertEquals($expected, $result);
     }
+
+    public function testNewSentence_UpdatesLastContributionField() {
+        $user = $this->Sentence->Users->get(1);
+        CurrentUser::store($user);
+        
+        Time::setTestNow(new Time('2019-02-01 00:00:00')); 
+        $this->Sentence->saveNewSentence('This is my new English sentence.', 'eng', 1);
+        $user = $this->Sentence->Users->get(1);
+        $previousLastContribution = $user->last_contribution;
+
+        Time::setTestNow(new Time('2019-02-02 00:00:00')); 
+        $this->Sentence->saveNewSentence('This is my newer English sentence.', 'eng', 1);
+        $user = $this->Sentence->Users->get(1);
+        $newLastContribution = $user->last_contribution;
+
+        $this->assertGreaterThan($previousLastContribution, $newLastContribution);
```

Comment:
This test works but it feels overkill. I think you can simplify it by only saving one sentence and checking that `users.last_contribution` field equals the time you set in `setTestNow()`.

Reply:
Also please call `Time::setTestNow()` without any parameter at the end of the test, otherwise the "now" time will be set in all subsequent tests, too.